src/client/serialize/SConscript

```diff
@@ -0,0 +1,29 @@
+"""Build DAOS Serialize Library"""
+import daos_build
+
+def scons():
+    """Execute build"""
+    Import('env')
+
+    denv = env.Clone()
+
+    have_hdf5 = True
+
+    # If hdf5 installed build DAOS serialization library
+    conf = Configure(denv)
+    if not conf.CheckLibWithHeader('hdf5', 'hdf5.h', 'c'):
+        have_hdf5 = False
+    conf.Finish()
+
+    libraries = ['daos', 'hdf5']
+
+    src = ['daos_serialize.c']
+    if have_hdf5 is True:
+        print("Building with hdf5 bindings.")
+        daos_serialize = daos_build.library(denv, 'daos_serialize', src,
```

Comment:
is there a circular dependency here similar to what we have with mpi-io ?

In other words, would be be the case that we may want to build client first, then hdf, then daos_serialize?  If so, we may want to look into using a build target for this build

Reply:
no there isn't.
this does not need the HDF5-VOL, but only HDF5 which are separate libraries.
you don't need DAOS to build HDF5. for this you can even use the HDF5 provided by the distribution